jenkins/databricks/build.sh

```diff
@@ -68,80 +68,122 @@ CUDF_JAR=${M2DIR}/ai/rapids/cudf/${CUDF_VERSION}/cudf-${CUDF_VERSION}-${CUDA_VER
 # pull normal Spark artifacts and ignore errors then install databricks jars, then build again
 JARDIR=/databricks/jars
 # install the Spark pom file so we get dependencies
-SQLJAR=----workspace_${SPARK_MAJOR_VERSION_STRING}--sql--core--core-hive-2.3__hadoop-2.7_${SCALA_VERSION}_deploy.jar
-CATALYSTJAR=----workspace_${SPARK_MAJOR_VERSION_STRING}--sql--catalyst--catalyst-hive-2.3__hadoop-2.7_${SCALA_VERSION}_deploy.jar
-ANNOTJAR=----workspace_${SPARK_MAJOR_VERSION_STRING}--common--tags--tags-hive-2.3__hadoop-2.7_${SCALA_VERSION}_deploy.jar
-COREJAR=----workspace_${SPARK_MAJOR_VERSION_STRING}--core--core-hive-2.3__hadoop-2.7_${SCALA_VERSION}_deploy.jar
+case "$BASE_SPARK_VERSION" in
+    "3.2.1")
+        COMMONS_LANG3_VERSION=3.12.0
+        COMMONS_IO_VERSION=2.8.0
+        DB_VERSION=-0003
+        FASTERXML_JACKSON_VERSION=2.12.3
+        HADOOP_VERSION=3.2
+        HADOOP_FULL_VERSION=3.2.
```

Comment:
Nice catch!  This was leftover from an attempt to reconcile jar dependencies where I thought we needed the full major.minor.patch version for the Hadoop jars, but then I ended up just pulling in the normal Hadoop client jar rather than worrying about using the exact version from the Databricks deploy.  If code compiled against the official Hadoop jars doesn't run on Databricks then that would be a real problem for that platform.